concourse/update-content/script.bash

```diff
@@ -43,21 +26,40 @@ for book_id in $book_ids; do
   git checkout src/config.books.js
   (git checkout "$branch" && git merge origin/master --no-edit -X theirs) || git checkout -b "$branch"
 
-  node script/entry update-content-versions --only "$book_id"
+  # there may be multiple approved versions for supporting different products, historical support, etc
+  approved_versions=$(jq -r --arg uuid "$book_id" '[.[]|select(.uuid==$uuid)][].version' <<< "$approved_books")
+
+  # in general we want the most recent one, in the future we'll have to filter this by which ones
+  # are available for the recipe code version REX supports
+  desired_version=$(sort --version-sort -r <<< "$approved_versions" | head -n 1)
+
+  current_version=$(jq -r --arg uuid "$book_id" '.[$uuid].defaultVersion' < src/config.books.json)
 
-  git add src/config.books.js
+  # approved book format has a "1." on the front of every version
+  desired_version=${desired_version:2}
+
+  if [ "$desired_version" == "$current_version" ]; then
+    echo "$book_id alredy at desired version."
+    continue
+  fi
+
+  jq \
+    --arg version "$desired_version" \
+    --arg uuid "$book_id" \
+    '.[$uuid].defaultVersion = $version' < src/config.books.json > src/config.books.json.new
+
+  mv src/config.books.json.new src/config.books.json
+
+  git add src/config.books.json
   git commit -m "update content" || true
   git push --set-upstream origin "$branch"
 
   book_title=$(node script/entry.js book-info "$book_id" --field title)
-
   curl -s -X POST -H "Authorization: token $GITHUB_ACCESS_TOKEN" "https://api.github.com/repos/openstax/rex-web/pulls" --data-binary @- << JSON
     {
       "title": "$book_title updates",
       "head": "$branch",
-      "base": "master",
-      "draft": true
+      "base": "master"
```

Comment:
Could/Should this look up the default branch? `default_branch` in https://api.github.com/repos/openstax/rex-web